Week7/week7 solution.py

```diff
@@ -53,34 +53,34 @@
 df = df.append(new_rows, ignore_index=True)
 df.reset_index(drop=True, inplace=True)
 
-5)add 2 row after 3rd row
+#5)add 2 row after 3rd row
 
 header=['Product No', 'Product Name', 'January', 'February', 'March', 'April', 'May', 'June','Total','Average']
 df.loc[2.5] = [15, 'cable',1290,7128,7456,4325,3874,2002,26075,4345.8333333]
 df = df.sort_index().reset_index(drop=True)
 df.loc[3.5] = [16, 'projecter',1140,1148,2260,2245,2256,2221,11270,1878.333333]
 df = df.sort_index().reset_index(drop=True)

```

Comment:
what is the use of drop=true?

Reply:
If you set drop = True , reset_index will delete the index instead of inserting it back into the columns of the DataFrame.